### DISCUSSION TITLE
Is there a threshold that can be set for disabling bridge on small surfaces ?

### DISCUSSION THREAD

**Discussion by DzzD** [supermerill/SuperSlicer#2096] [Q&A] [answered]
I made a simple model to test fan blowing into overhangs, and notice that there is a useless bridge, I looked everywhere but cant find a way to disable bridge (like a min surface) ?

![image](https://user-images.githubusercontent.com/12198339/146782490-f0e68a87-7bb0-40c6-9f4f-0df613679c6d.png)

![image](https://user-images.githubusercontent.com/12198339/146782606-0b2d5d08-ce8a-472f-9ac7-31c1f79219b0.png)

**Comment by neophyl** [comment, marked as answer]
Thats bridge infill, above the degree symbol which is basically doesn't have room to do a top infill as its so close.  

As far as I know there isn't any way to completely turn off bridge infill.  Be happy to learn otherwise though :)  If it bothers you then add a bit of variable layer height to the top to make the layers thinner.  All the pretty pointless ironing would bug me more though.  I shudder to think what that's doing to the print time and also the risk of having an extruder clog.

That might be a nice feature request though, to always make sure that the top layer is top infill rather than bridge infill when there isn't enough room.

**Reply by DzzD** [reply]
> That might be a nice feature request though, to always make sure that the top layer is top infill rather than bridge infill when there isn't enough room.

Sure, it might be also useful on small surface because bridging very small distances/surface have no real interest, it is IMHO not required to use bridge algorithm on small surfaces or distances.

About ironing : I always set top infill as ironing (not post process) as I found this make top surface a lot nicer with not that much more time (also I always use a 0.6mm nozzle and never had a clog and plain surface printing is nearly 50% faster)

**Reply by DzzD** [reply]
> If it bothers you then add a bit of variable layer height to the top to make the layers thinner

Thanks for the tip, I just tried and it works. But my question was more for general purpose than for this particular model

**Reply by supermerill** [reply]
yes, not implemented, but I think there is still a hardcoded one with something like a perimeter width.
